datastore/ac-happy-home-designer/protocol.go

```diff
@@ -0,0 +1,152 @@
+// Package protocol implements the Animal Crossing: Happy Home Designer protocol
+package protocol
+
+import (
+	"fmt"
+	"slices"
+
+	"github.com/PretendoNetwork/nex-go/v2"
+	"github.com/PretendoNetwork/nex-go/v2/types"
+	datastore "github.com/PretendoNetwork/nex-protocols-go/v2/datastore"
+	datastore_ac_happy_home_designer_types "github.com/PretendoNetwork/nex-protocols-go/v2/datastore/ac-happy-home-designer/types"
+	"github.com/PretendoNetwork/nex-protocols-go/v2/globals"
+)
+
+const (
+	// ProtocolID is the Protocol ID for the DataStore (Animal Crossing: Happy Home Designer) protocol. ID is the same as the DataStore protocol
+	ProtocolID = 0x73
+
+	// MethodGetObjectInfos is the method ID for GetObjectInfos
+	MethodGetObjectInfos = 0x2D
+
+	// MethodGetMetaByOwnerId is the method ID for GetMetaByOwnerId
+	MethodGetMetaByOwnerId = 0x2E
+
+	// MethodGetMetaByUniqueId is the method ID for GetMetaByUniqueId
+	MethodGetMetaByUniqueId = 0x2F
```

Comment:
```suggestion
	// MethodGetMetaByOwnerID is the method ID for GetMetaByOwnerID
	MethodGetMetaByOwnerID = 0x2E

	// MethodGetMetaByUniqueID is the method ID for GetMetaByUniqueID
	MethodGetMetaByUniqueID = 0x2F
```

Same for every ocurrence